### DISCUSSION TITLE
run naval_fate

### DISCUSSION THREAD

**Discussion by povociancioni** [docopt/docopt.net#167] [General]
[Program.txt](https://github.com/docopt/docopt.net/files/5683102/Program.txt)
Hello,
Why is this Program.cs version of .txt not permitting to run the application yet?

Thank you.

Patrick

**Comment by iwillspeak** [comment]
Your example seems to be missing a declaration of `usage`. You probably want this to be a `const string` Containing your Docopt usage text.

**Comment by povociancioni** [comment]
Yes.  I find it a long way that I still have to go ...
I'm achieving very few and moreover very slowly, though I'm not faking. I have small experience of computer propramming and still am awaiting a faster tool (or I wish I learned faster) especially when I'm comparing to what I got... Thank you.
(I got :)
-h = False
-s = False
-o = False

FILE
--quiet = False
--verbose =False
INPUT = [ ]
